File: problemSets/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

void hashm(int i, int n);

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        for (int k = n - i - 1; k > 0; k = k - 1)
        {
            printf(" ");
        }
        hashm(i, n);
        printf("  ");
        hashm(i, n);
        printf("\n");
    }
}

void hashm(int i, int n)
{
    for (int j = 0; j <= i; j++)
    {
        printf("#");
    }
}

File: problemSets/pset6/mario/more/mario.py
import cs50


def blocks(i, h):
    for j in range(i + 1):
        print('#', end='')


def main():
    h = 0
    while h < 1 or h > 8:
        h = cs50.get_int("Height: ")

    for i in range(h):
        for k in range(h - i - 1, 0, - 1):
            print(' ', end='')

        blocks(i, h)
        print('  ', end='')
        blocks(i, h)
        print('\n', end='')


main()